cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaAvailabilityTest.java

```diff
@@ -274,17 +262,17 @@ Admin ac() {
     @Test
     public void testBelowMinIsr(VertxTestContext context) {
         KSB ksb = new KSB()
-            .addNewTopic("A", false)
-                .addToConfig(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG, "2")
-                .addNewPartition(0)
+            .addNewTopic("A")
+                .addToConfig("2")
```

Comment:
Wasn't it much more readable before when it was clear what the `2` meant? It is called `addToConfig` to add configuration to the topic. It is not called `setMinInSyncReplicas`. So I think this is wrong.

Reply:
Yep, this is a definite revert.